spec/domain/queries/find_content_spec.rb

```diff
@@ -156,4 +262,51 @@
       expect(-> { described_class.call(filter: filter) }).to raise_error(ArgumentError)
     end
   end
+
+  describe '#select_view' do
```

Comment:
As per my comment below I would extract this into the unit tests for QueryPlanner.

Reply:
👍